application/features/bootstrap/IdpIdBrokerIntegrationContext.php

```diff
@@ -0,0 +1,332 @@
+<?php
+namespace Sil\Idp\IdSync\Behat\Context;
+
+use Behat\Behat\Tester\Exception\PendingException;
+use Behat\Behat\Context\Context;
+use Exception;
+use PHPUnit\Framework\Assert;
+use Sil\Idp\IdBroker\Client\IdBrokerClient;
+use Sil\Idp\IdSync\common\components\adapters\IdpIdBroker;
+use Sil\PhpEnv\Env;
+
+/**
+ * Defines application features from the specific context.
+ */
+class IdpIdBrokerIntegrationContext implements Context
+{
+    /**
+     * @var IdpIdBroker
+     */
+    protected $idBroker;
+    
+    protected $oldPassword;
+    protected $newPassword;
+    
+    protected $testUserData;
+    protected $testUpdatedUserData;
+    
+    protected $result;
+    
+    public function __construct()
+    {
+        require_once __DIR__ . '/../../vendor/yiisoft/yii2/Yii.php';
+        
+        $this->idBroker = new IdpIdBroker([
+            'baseUrl' => 'http://broker', // For tests. Matches docker container name.
+            'accessToken' => Env::requireEnv('ID_BROKER_ACCESS_TOKEN'),
+        ]);
+        
+        $this->testUserData = $this->generateDataForNewTestUser();
+    }
+    
+    protected function generateDataForNewTestUser()
+    {
+        $uniqueId = uniqid();
+        return [
+            'employee_id' => (string)$uniqueId,
+            'first_name' => 'Test',
+            'last_name' => 'User',
+            'display_name' => 'Test User',
+            'username' => 'user' . $uniqueId,
+            'email' => 'user' . $uniqueId . '@example.com',
+        ];
+    }
+    
+    protected function generateDummyPassword()
+    {
+        return base64_encode(random_bytes(12));
+    }
+
+    /**
+     * @Given a user exists
+     */
+    public function aUserExists()
+    {
+        $newUser = $this->idBroker->createUser($this->testUserData);
+        Assert::assertNotNull($newUser);
+    }
+
+    /**
+     * @Given that user is not active
+     */
+    public function thatUserIsNotActive()
+    {
+        // Deactivate the user.
+        $this->idBroker->deactivateUser(
+            $this->testUserData['employee_id']
```

Comment:
Since you use this so often, your constructor could add an attribute for $this->testEmployeeId.